sdcm/nemesis.py

```diff
@@ -583,9 +583,35 @@ def _run_in_cqlsh(self, cmd, node=None):
 
         node.remoter.run('cqlsh {} -e "{}" {}'.format(cql_auth, cmd, node.private_ip_address), verbose=True)
 
-    def _modify_table_property(self, name, val, keyspace="keyspace1", table="standard1"):
+    def select_keyspace_and_table_for_nemesis(self, select_table=False):
```

Comment:
I would call it set_target_table (like we have set_target_node)

Reply:
the function is removed